src/ids-header/ids-header.js

```diff
@@ -0,0 +1,73 @@
+import {
+  IdsElement,
+  customElement,
+  scss,
+  mix,
+  attributes
+} from '../ids-base/ids-element';
+
+import {
+  IdsEventsMixin,
+  IdsKeyboardMixin,
+  IdsThemeMixin
+} from '../ids-mixins';
+
+import styles from './ids-header.scss';
+import '../ids-input/ids-input';
```

Comment:
```suggestion
import IdsInput from '../ids-input/ids-input';
```

Same for all these imports... But sort of wondering if we want to require all these. @EdwardCoyle ?